pages/docs/installation/installation-source-dependencies.md

```diff
@@ -262,6 +262,10 @@ If you want build preCICE on your own computer and you are using one of the foll
 
 Other modern versions of popular Linux distributions are also perfectly compatible, here we just list a few popular options. Since our users have tried preCICE on various distributions, you may as well ask on our [forum](https://precice.discourse.group/) for any questions.
 
+### Ubuntu 22.04 LTS Jammy Jellyfish
+
+Follow the instructions as for Ubuntu 20.04.
```

Comment:
It seems a bit weird to have 22.04 described as "same as 20.04" instead of the other way around it.

Reply:
Good point, I swapped them. Thanks!